lib/github_oauthable.rb

```diff
@@ -27,6 +27,8 @@ module GitHubOAuthable
   GRAPHQL
 
   included do
+    before_action :set_error_context_user if respond_to?(:before_action)
```

Comment:
What about to put this into `Admin::AdminController` instead?

Reply:
Because that is not included in the Avo base controller

Reply:
OK, I'll check how Avo works. I thought `Admin::AdminController` is going to be new base controller for whole namespace.